FrontEnd/src/components/CheckBoxCustom.js

```diff
@@ -10,7 +10,7 @@ class CheckBoxCustom extends React.Component {
   render() {
     return (
       
-      <div class="check-box">
+      <div className="check-box">
```

Comment:
did you checked if this is not clashing with the framework checkbox?

Reply:
it doesn't appear to be? I don't think they put a dash in it but i'm not sure